Backpack/src/main/java/net/skyscanner/backpack/button/BpkButton.kt

```diff
@@ -32,23 +32,26 @@ import net.skyscanner.backpack.button.internal.ButtonStyles
 import net.skyscanner.backpack.button.internal.ICON_POSITION_END
 import net.skyscanner.backpack.button.internal.ICON_POSITION_ICON_ONLY
 import net.skyscanner.backpack.button.internal.ICON_POSITION_START
+import net.skyscanner.backpack.text.BpkText
 import net.skyscanner.backpack.util.unsafeLazy
 import net.skyscanner.backpack.util.use
+import kotlin.math.roundToInt
 
 open class BpkButton(
   context: Context,
   attrs: AttributeSet?,
   defStyleAttr: Int,
   type: Type,
+  private val format: BpkButtonFormat = BpkButtonFormat.Small,
 ) : BpkButtonBase(context, attrs, defStyleAttr) {
 
-  constructor(context: Context) : this(context, null, 0, Type.Primary)
+  constructor(context: Context) : this(context, null, 0, Type.Primary, BpkButtonFormat.Small)
```

Comment:
BpkButtonSize causes compilation error, as some of the libraries has app:buttonSize property already.